context/app/static/js/components/Search/config.js

```diff
@@ -18,20 +19,28 @@ function makeDonorMetadataFilters(isDonor) {
 
 const affiliationFilters = [listFilter('group_name', 'Group'), listFilter('created_by_user_displayname', 'Creator')];
 
+const sharedTileFields = [
+  field('last_modified_timestamp', 'Last Modified Unmapped'),
+  field('descendant_counts.entity_type', 'Descendant Counts'),
+];
+
 const donorConfig = {
   filters: {
     'Donor Metadata': makeDonorMetadataFilters(true),
     Affiliation: affiliationFilters,
   },
-  fields: [
-    field('display_doi', 'Donor'),
-    field('group_name', 'Group'),
-    field(`mapped_metadata.${ageField}`, 'Age'),
-    field(`mapped_metadata.${bmiField}`, 'BMI'),
-    field('mapped_metadata.sex', 'Sex'),
-    field('mapped_metadata.race', 'Race'),
-    field('mapped_last_modified_timestamp', 'Last Modified'),
-  ],
+  fields: {
+    table: [
+      field('display_doi', 'Donor'),
+      field('group_name', 'Group'),
+      field(`mapped_metadata.${ageField}`, 'Age'),
+      field(`mapped_metadata.${bmiField}`, 'BMI'),
+      field('mapped_metadata.sex', 'Sex'),
+      field('mapped_metadata.race', 'Race'),
+      field('mapped_last_modified_timestamp', 'Last Modified'),
+    ],
+    tile: [...sharedTileFields, field(`mapped_metadata.${ageUnitField}`, 'Age Unit')],
+  },
```

Comment:
Would it be simpler just always get a superset of fields that will work for both UIs? I don't think over-fetching is a big concern, and I wouldn't want to add complexity to the code just to make the requests a little smaller.

But not a big deal: Your call.

Reply:
I think the way it is setup we are over-fetching which I agree isn't a big concern. The reason why these are kept separately is to be able to pass the specific fields needed to `ResultsTable` and `SortingTableHead`. Maybe I should rename the variables to make more sense?

Reply:
So we could keep a resultsFields superset and also a tableFields variable to inform `ResultsTable` and `SortingTableHead`?

Reply:
Per our Zoom conversation, the way this is now handled is fine, but worth monitoring if things get more complex in the future.